src/core/searchUtils.js

```diff
@@ -35,6 +41,14 @@ export function convertQueryParamsToFilters(params) {
   }, {});
 }
 
+export function convertOperatingSystemToFilterName(name) {
```

Comment:
I think this is actually `convertOperatingSystemToFilterValue`, right? Also, I think abbreviating it to `OS` would make the code a bit easier to read but no biggie.

Reply:
Yes and yes. 👍